docs/src/main/sphinx/release/release-436.md

```diff
@@ -0,0 +1,57 @@
+# Release 436 (10 Jan 2024)
+
+## General
+
+* {{breaking}} Require JDK 21.0.1 to run Trino. ({issue}`20010`)
+* Improve performance by not generating redundant predicates. ({issue}`16520`)
+* Fix query failure when invoking the `json_table` function. ({issue}`20122`)
+* Fix query hang when a [SQL routine](/routines) dereferences a row field. ({issue}`19997`).
+* Fix potential incorrect results when using the {func}`ST_Centroid` and
+  {func}`ST_Buffer` functions for tiny geometries. ({issue}`20237`)
+
+## Delta Lake connector
+
+* Add the `parquet_ignore_statistics` catalog session property, which can be
+  enabled to query files with corrupted or incorrect statistics. ({issue}`20228`)
+* Improve performance of queries with selective joins on partition columns. ({issue}`20261`)
+* Reduce the number of requests made to AWS Glue when listing tables, schemas,
+  or functions. ({issue}`20189`)
+* Fix incorrect results when querying Parquet files containing column indexes
+  when the query has filters on multiple columns. ({issue}`20267`)
+
+## Hive connector
+
+* Allow reading 3-element unions when the sub-union is composed of compatible
+  types or nulls. ({issue}`20233`)
+* Reduce the number of requests made to AWS Glue when listing tables, schemas,
+  or functions. ({issue}`20189`)
+* Fix incorrect results when querying Parquet files containing column indexes
+  when the query has filters on multiple columns. ({issue}`20267`)
+
+## Hudi connector
+
+* Add support for enforcing that a filter on a partition key must be present in
+  the query. This can be enabled by with the
+  ``hudi.query-partition-filter-required`` configuration property or the
+  ``query_partition_filter_required`` catalog session property. ({issue}`19906`)
+* Fix incorrect results when querying Parquet files containing column indexes
+  when the query has filters on multiple columns. ({issue}`20267`)
+
+## Iceberg connector
+
+* Add the `parquet_ignore_statistics` catalog session property. ({issue}`20228`)
+* Improve performance of queries with selective joins on partition columns. ({issue}`20212`)
+* Reduce the number of requests made to AWS Glue when listing tables, schemas,
+  or functions. ({issue}`20189`)
+* Fix potential loss of data when running multiple `INSERT` queries at the same
+  time ({issue}`20093`).
+* Fix incorrect results when providing a nonexistent namespace while listing
+  namespaces. ({issue}`19980`)
+
+## OpenSearch connector
+
+* Add [OpenSearch connector](/connector/opensearch). ({issue}`11377`)
```

Comment:
Typically we do that in general when a connector is first added .. also it might be worth adding it in the Elasticsearch connector section instead?